src/Shared/Components/TabGroup/TabGroup.component.tsx

```diff
@@ -32,24 +33,38 @@ const Tab = ({
     const getTabComponent = () => {
         const content = (
             <>
-                {showError && <ICErrorExclamation className={`${iconClassName}`} />}
-                {!showError && showWarning && <ICWarning className={`${iconClassName} warning-icon-y7`} />}
-                {!showError && !showWarning && Icon && (
-                    <Icon className={`${iconClassName} ${getIconColorClassMap({ active })[iconType] || ''}`} />
-                )}
-                {label}
-                {badge !== null && (
-                    <div className={`tab-group__tab__badge bcn-1 cn-7 fw-6 flex px-4 ${badgeClassName}`}>{badge}</div>
+                <p className="m-0 flexbox dc__align-items-center dc__gap-6">
```

Comment:
We should extract this in a function so that condition cluttering can be removed